src/AudioAnalysisTools/CommonParameters.cs

```diff
@@ -95,8 +101,8 @@ public abstract class CommonParameters : IValidatableObject
 
         public virtual IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
         {
-            yield return this.MinHertz.ValidateNotNull(nameof(this.MinHertz));
-            yield return this.MaxHertz.ValidateNotNull(nameof(this.MaxHertz));
+            //yield return this.MinHertz.ValidateNotNull(nameof(this.MinHertz));
+            //yield return this.MaxHertz.ValidateNotNull(nameof(this.MaxHertz));
```

Comment:
Is there a reason this validation is disabled?

Revert change or delete validation (and explain).